src/main/java/Duke.java

```diff
@@ -1,10 +1,160 @@
+                "____________________________________________________________\n";
+    }
+
+    public static String printOutro() {
+        return "____________________________________________________________\n" +
+                " Bye. Hope to see you again soon!\n" +
+                "____________________________________________________________\n";
+    }
+
+    public static String list(ArrayList<Task> a, int counter) {
+        String x = ("Here are the tasks in your list: " + "\n" +
+                "____________________________________________________________\n");
+        String result = "";
+        for (int i = 0; i < counter; i++) {
+            result += (i + 1) + ". " + a.get(i).toString() + "\n";
+        }
+        String y = ("____________________________________________________________\n");
+        return x + result + y;
+    }
+
+    public static String printMark(Task task) {
+        if (task.getStatus()) {
+            return "____________________________________________________________\n" +
+                    "Nice! I've marked this task as done:" + "\n" + task.toString() + "\n" +
+                    "____________________________________________________________\n";
+        } else {
+            return "____________________________________________________________\n" +
+                    "OK, I've marked this task as not done yet-:" + "\n" + task.toString() + "\n" +
+                    "____________________________________________________________\n";
+        }
+    }
+
+    public static String added(Task task, int count) {
+        return "____________________________________________________________\n" +
+                "Got it.I've added this task:" + "\n" + task.toString() + "\n" +
+                "Now you have " + count + " tasks in the list." + "\n" +
+                "____________________________________________________________\n";
+    }
+
+    public static String getTask(String message) {
+        int start = message.indexOf(" ") + 1;
+        int end = message.indexOf("/");
+        if (end == -1) {
+            // "/" not found, handle the case, e.g., throw an exception or return the rest of the string
+            end = message.length();
+        }
+        String task = message.substring(start, end);
+        return task.trim();
+    }
+
+
+    public static String getTime(String message) {
+        int start = message.indexOf("/");
+        String time = message.substring(start + 1);
+        return time;
+    }
+
+    public static String deleteMessage(Task task, int count) {
+        return "____________________________________________________________\n" +
+                "Noted. I've removed this task:" + "\n" + task.toString() + "\n" +
+                "Now you have " + count + " tasks in the list." + "\n" +
+                "____________________________________________________________\n";
+    }
+
+    public static ArrayList<Task> delete(ArrayList<Task> a, int count) {
```

Comment:
The method name could be changed to explain more about what this method is deleting.